tests/mod_api/test_models.py

```diff
@@ -141,20 +148,33 @@ def test_return_none_if_object_found(self):
     @mock_s3
     def test_change_acl(self):
         with self.app.app_context():
-            bit_store = BitStore('test_pub', 'test_package')
+            public_grants = ['READ', 'FULL_CONTROL']
+            private_grants = ['FULL_CONTROL']
+
+            bit_store = BitStore('test_pub', 'test_package', body='test')
             s3 = boto3.client('s3')
             bucket_name = self.app.config['S3_BUCKET_NAME']
             s3.create_bucket(Bucket=bucket_name)
-            read_me_key = bit_store.build_s3_key('test.md')
-            data_key = bit_store.build_s3_key('data.csv')
             metadata_key = bit_store.build_s3_key('datapackage.json')
-            s3.put_object(Bucket=bucket_name, Key=read_me_key, Body='')
-            s3.put_object(Bucket=bucket_name, Key=data_key, Body='')
-            s3.put_object(Bucket=bucket_name, Key=metadata_key, Body='')
+
+            bit_store.save_metadata()
+
+            res = s3.get_object_acl(Bucket=bucket_name, Key=metadata_key)
+
+            for grant in res['Grants']:
+                self.assertTrue(grant['Permission'] in public_grants)
```

Comment:
@Fak3 it is not the admin user. It is moto generated ```ID```. Look at this code
```python
owner_grantee = FakeGrantee(id='75aa57f09aa0c8caeab4f8c24e99d10f8e7faeebf76c078efc7c6caea54ba06a')
``` 
so if they change the id in future, It will fail.